### PR TITLE
Patch version 4.61.1

### PR DESCRIPTION
**Pull request by northwestwitch**
Before merging more extensive test demanding features (like the joblib) I was thinking that we could release a patch with the easy fixes to the ClinVar downloads, so they are done and out of the way.

### Fixed
- [x] Added `UMLS` as an option of `Condition ID type` in ClinVar Variant downloaded files
- [x] Missing value for `Condition ID type` in ClinVar Variant downloaded files
- [x] Possibility to open, close or delete a ClinVar submission even if it doesn't have an associated name
- [x] Save SV type, ref and alt n. copies to exported ClinVar files
- [x] Inner and outer start and stop SV coordinates not exported in ClinVar files
- [x] ClinVar submissions page crashing when SV files don't contain breakpoint exact coordinates
- [x] Align OMIM diagnoses with delete diagnosis button on case page
- [x] In ClinVar form, reset condition list and customize help when condition ID changes

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,10 +4,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
-## [unreleased]
-### Added
-- Added `UMLS` as an option of `Condition ID type` in ClinVar Variant downloaded files
```
Comment (northwestwitch) [inline CHANGELOG.md:9]: This was actually a fix, because the UMLS value was missing

**Comment by northwestwitch** [comment]
Something's still wrong with the parsing of condition ID:

image

**Comment by northwestwitch** [comment]
> Something's wrong with the parsing of condition ID Value:
> 
> image

Ah no, it works, I just didn't click when entering the value!

**Comment by dnil** [comment]
Sure, I tested again, and have two small suggestions related to this patch. Doesn't have to be now, but might as well?

https://github.com/Clinical-Genomics/scout/issues/3699
https://github.com/Clinical-Genomics/scout/issues/3700

**Comment by northwestwitch** [comment]
> Sure, I tested again, and have two small suggestions related to this patch. Doesn't have to be now, but might as well?
> 
> #3699 #3700

Nice. They are easy things, let's include them!

**Review by dnil — APPROVED** [review]

**Pull request merged**